### PR TITLE
revert PR 22489

### PR DESCRIPTION
**Pull request by wangleis**
### Details:
 - *revert PR 22489*

### Tickets:
 - *CVS-129030*

### CODE REVIEW

**Review by dmitry-gorokhov — APPROVED** [review]

File: src/tests/test_utils/functional_test_utils/layer_tests_summary/run_parallel.py
Comment (iefode) [inline src/tests/test_utils/functional_test_utils/layer_tests_summary/run_parallel.py:43]: @wangleis @dmitry-gorokhov 
I will ask you not to do such changes in such utils without review from ie test infra maintainers. Such changes can affect GHA CI
CC @andrei-kochin

Comment (wangleis) [reply src/tests/test_utils/functional_test_utils/layer_tests_summary/run_parallel.py:43]: @iefode [PR 22489](https://github.com/openvinotoolkit/openvino/pull/22489) is reverted in this PR and replaced by [PR 23099](https://github.com/openvinotoolkit/openvino/pull/23099) which did not change GHA CI setting. Thanks.

Comment (iefode) [reply src/tests/test_utils/functional_test_utils/layer_tests_summary/run_parallel.py:43]: FYI: this script is used to run func tests in GHA. It is the main reason why I asked you do not make such changes without approve

**Pull request merged**
